### PR TITLE
RapiDoc added and tested.

### PR DESCRIPTION
**Pull request by webdastur**

### CODE REVIEW

**Comment by webdastur** [comment]
@vitalik Hi Vitaliy. I added RapiDoc and tested it locally, production. Please check this. Thanks.

**Comment by webdastur** [comment]
@vitalik Is there anything I should add to this PR for approval?

File: ninja/openapi/views.py [hunk 1]
```diff
@@ -16,12 +16,16 @@
 __all__ = ["default_home", "openapi_json", "openapi_view", "openapi_view_cdn"]
 
 
-render_swagger = ninja_settings.DOCS_VIEW == "swagger"
-view_tpl = "ninja/swagger.html" if render_swagger else "ninja/redoc.html"
+render_doc = "swagger"
+
+if ninja_settings.DOCS_VIEW == "redoc":
```
Comment (baseplate-admin) [inline ninja/openapi/views.py:21]: Can we use a walrus operator here?

**Review by baseplate-admin — CHANGES_REQUESTED** [review]

**Comment by vitalik** [comment]
> @vitalik Is there anything I should add to this PR for approval?

well, I'm thinking about some more flexible solution so that anyone can add their favourite docs engine...

File: ninja/openapi/views.py (hunk 1 above)
Comment (vitalik) [reply ninja/openapi/views.py:21]: > Can we use a walrus operator here?

no - since we support pytthon3.7 and pytrhon3.6 probably works as well

**Comment by vitalik** [comment]
Hi @webdastur 


since django-ninja v1 allows to make plugable openapi viewers

```Python
api = NinjaAPI(docs=YourEngine())
```

see how redoc implemented here https://github.com/vitalik/django-ninja/blob/2f5a5766ea6e86784d5eb5cafc0ab3e0e5f60e96/ninja/openapi/docs.py#L58-L77

you can make a separate package for Elements - and we can promote it in the docs

**Pull request closed without merging**